### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by frechg**

### CODE REVIEW

File: app/views/accounts/show.haml [hunk 1]
```diff
@@ -4,12 +4,7 @@
 
 %section.account-section
   %aside.account-nav
-    %p.message
-      If you'd like to pay for Hound in bulk on a monthly or annual
-      basis we're happy to make that happen. Email
-      #{mail_to "hound@thoughtbot.com"}
-      with your GitHub org name and the number of repos you'd like to
-      activate.
+    %p.message= t('account.bulk_pricing', mail_to: mail_to('hound@thoughtbot.com') ).html_safe
```
Comment (gylaz) [inline app/views/accounts/show.haml]: How about using double quotes?

Comment (gylaz) [inline app/views/accounts/show.haml]: Also there's a space before the last paren.

File: config/locales/en.yml [hunk 2]
```diff
@@ -7,6 +7,14 @@ en:
   pending_status: "Hound is busy reviewing changes..."
   success_status: "Hound has reviewed all the changes!"
 
+  account:
+    bulk_pricing:
+      "If you'd like to pay for Hound in bulk on a monthly or annual
+      basis we're happy to make that happen. Email
+      %{mail_to}
```
Comment (gylaz) [inline config/locales/en.yml]: Can this be moved to the end of the previous line?

**Comment by gylaz** [comment]
A few comments. :+1:

**Comment by salbertson** [comment]
@frechg, what's left for this to be ready to merge?

**Pull request merged**